apps/web/src/blocks/HeroBlock/HeroBlock.config.ts

```diff
@@ -53,6 +25,49 @@ const HeroBlock: Block = {
       label: 'Eyebrow',
       required: false
     },
+    {
+      name: 'layout',
+      label: 'Layout',
+      required: false,
+      type: 'select',
+      options: [
+        {
+          label: 'Content on Right',
+          value: 'contentRight'
+        },
+        {
+          label: 'Content on Left',
+          value: 'contentLeft'
+        },
+        {
+          label: 'Content Centered',
+          value: 'contentCenter'
+        }
+      ]
+    },
+    {
+      name: 'contentAlign',
+      label: 'Content Alignment',
+      required: false,
+      type: 'select',
+      admin: {
+        description: 'If copy flows from left, right, or center.'
+      },
+      options: [
+        {
+          label: 'Align Right',
+          value: 'right'
+        },
+        {
+          label: 'Align Left',
+          value: 'left'
+        },
+        {
+          label: 'Align Center',
+          value: 'center'
+        }
+      ]
+    },
     {
       type: 'text',
       name: 'title',
```

Comment:
Let's update this so that the title and subtitle are handled in the **same** richText `content` field